src/components/AlbumInfo/AlbumInfo.tsx

```diff
@@ -93,8 +96,10 @@ export default function AlbumInfo({ collection }: AlbumInfoProps): JSX.Element {
             </span>
           ))}
         </div>
-        {collection.styles.length > 0 ? <h3>Style:</h3> : null}
-        {collection.styles.length > 0 ? (
+        {collection.styles && collection.styles.length > 0 ? (
+          <h3>Style:</h3>
+        ) : null}
+        {collection.styles && collection.styles.length > 0 ? (
```

Comment:
you use collection.styles very often. can you destructure it anywhere? ` const {styles} = collection`